PPAK_CVS/Sources/Storage/Storage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Storage.swift
+//  PPAK_CVS
+//
+//  Created by 김응철 on 2022/12/25.
+//
+
+import Foundation
+
+final class Storage {
+  
+  static let shared = Storage()
+  
+  private let key = "Storage"
+  private let userDefaults = UserDefaults.standard
+  private init() {}
+  
+  private(set) var products: [ProductModel] = shared.load() {
+    didSet {
+      let encoder = JSONEncoder()
+      if let encoded = try? encoder.encode(oldValue) {
+        userDefaults.set(encoded, forKey: key)
+      }
```

Comment:
한 번만 사용되는 인스턴스라면 변수에 저장하지 않고 바로 호출하여 접근하는 게 적절할 것 같아요.
```suggestion
    didSet {
      if let encoded = try? JSONEncoder().encode(oldValue) {
        userDefaults.set(encoded, forKey: key)
      }
```